arbeitszeit_web/www/presenters/select_productive_consumption_presenter.py

```diff
@@ -0,0 +1,68 @@
+from dataclasses import dataclass
+
+from arbeitszeit.records import ConsumptionType
+from arbeitszeit.use_cases.select_productive_consumption import (
+    InvalidPlanResponse,
+    NoPlanResponse,
+    ValidPlanResponse,
+)
+from arbeitszeit_web.notification import Notifier
+from arbeitszeit_web.translator import Translator
+
+
+@dataclass
+class SelectProductiveConsumptionPresenter:
+    notifier: Notifier
+    translator: Translator
+
+    @dataclass
+    class ViewModel:
+        valid_plan_selected: bool
+        plan_id: str | None
+        plan_name: str | None
+        plan_description: str | None
+        amount: int | None
+        is_consumption_of_fixed_means: bool
+        status_code: int
```

Comment:
From my POV, ideally, the type of **status_code** would be an enumeration type, where the names of the enumerators would be the standard meanings (OK, NOT_FOUND, etc.), and the values would be there standard values.  The benefit of this is clearer self-documenting code.  We should actually define such an enumeration type so that it is globally available.  But I can understand why you might not wish to bother.

Reply:
I prefer not to add to this already big PR a structural change. But I agree that this is not ideal as it is.